inference-engine/ie_bridges/java/README.md

```diff
@@ -0,0 +1,12 @@
+## Software Requirements 
+- openjdk 11
+
+### Linux
+To install openjdk use: 
+```bash
+sudo apt-get install -y openjdk-11-jdk
```

Comment:
On U16 I don't have such package. I have only openjdk-9-jdk. Can we extend this guide for different OSes? E.g. Ubuntu family, MacOSX, Windows.

Reply:
Even I've have installed java jdk, I have :
```
-- Found Java: /usr/bin/java (found version "9-internal")
CMake Error at /usr/local/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find JNI (missing: JAVA_INCLUDE_PATH JAVA_INCLUDE_PATH2)
Call Stack (most recent call first):
  /usr/local/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  /usr/local/share/cmake-3.16/Modules/FindJNI.cmake:372 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  inference-engine/ie_bridges/java/CMakeLists.txt:12 (find_package)


-- Configuring incomplete, errors occurred!
```

Reply:
Is that OK to write for Ubuntu 18.04 / 16.04 first? Like


## Software Requirements 

* Ubuntu 18.04
```
sudo apt-get install -y openjdk-11-jdk
```
* Ubuntu 16.04
```
sudo apt-get install -y openjdk-9-jdk
```

Reply:
@dkurt sure, it's OK.